src/app_service/service/token/service.nim

```diff
@@ -21,6 +21,14 @@ logScope:
 
 include async_tasks
 
+const ETHEREUM_SYMBOL = "ETH"
+const CRYPTO_SUB_UNITS_TO_FACTOR = {
+  "WEI": (ETHEREUM_SYMBOL, 1e-18),
```

Comment:
this is only for ETH that u use this? as for USDC, there is only 6 decimals

Reply:
This is just a way of using all the same currency formatting mechanism for gas fees, which use "Gwei" and "wei" as unit.